metis-media-service/src/main/java/eu/europeana/metis/mediaservice/WebResource.java

```diff
@@ -0,0 +1,166 @@
+package eu.europeana.metis.mediaservice;
+
+import java.math.BigInteger;
+import java.util.List;
+import java.util.Locale;
+import java.util.stream.Collectors;
+
+import eu.europeana.corelib.definitions.jibx.AudioChannelNumber;
+import eu.europeana.corelib.definitions.jibx.BitRate;
+import eu.europeana.corelib.definitions.jibx.CodecName;
+import eu.europeana.corelib.definitions.jibx.ColorSpaceType;
+import eu.europeana.corelib.definitions.jibx.DoubleType;
+import eu.europeana.corelib.definitions.jibx.Duration;
+import eu.europeana.corelib.definitions.jibx.HasColorSpace;
+import eu.europeana.corelib.definitions.jibx.HasMimeType;
+import eu.europeana.corelib.definitions.jibx.Height;
+import eu.europeana.corelib.definitions.jibx.HexBinaryType;
+import eu.europeana.corelib.definitions.jibx.IntegerType;
+import eu.europeana.corelib.definitions.jibx.LongType;
+import eu.europeana.corelib.definitions.jibx.NonNegativeIntegerType;
+import eu.europeana.corelib.definitions.jibx.OrientationType;
+import eu.europeana.corelib.definitions.jibx.SampleRate;
+import eu.europeana.corelib.definitions.jibx.SampleSize;
+import eu.europeana.corelib.definitions.jibx.SpatialResolution;
+import eu.europeana.corelib.definitions.jibx.StringType;
+import eu.europeana.corelib.definitions.jibx.Type1;
+import eu.europeana.corelib.definitions.jibx.WebResourceType;
+import eu.europeana.corelib.definitions.jibx.Width;
+
+public class WebResource {
```

Comment:
I think some documentation in this class would be nice? Just explaining why this class is here (instead of just using WebResourceType directly).